apps/poker-roll/src/app/homePage.jsx

```diff
@@ -1,12 +1,40 @@
 import React from "react";
-
+import Outlet from "react-router-dom";
+import Box from '@mui/material/Box';
+import BottomNavigation from '@mui/material/BottomNavigation';
+import BottomNavigationAction from '@mui/material/BottomNavigationAction';
+import EmojiEventsIcon from '@mui/icons-material/EmojiEvents';
+import TableBarIcon from '@mui/icons-material/TableBar';
+import PaidIcon from '@mui/icons-material/Paid';
 
 const Home = (props) => {
 
     return (
         <div>
-            <div></div>
+            <div className="header">
+                <h1>Poker Roll</h1>
+            </div>
+            <div>
+                <Outlet />
+            </div>
+            <div className="footer">
+            <Box sx={{ width: 500 }}>
+      <BottomNavigation
+        showLabels
+        value={value}
+        onChange={(event, newValue) => {
+          setValue(newValue);
+        }}
+      >
+        <BottomNavigationAction label="BankRoll" icon={<PaidIcon />} />
+        <BottomNavigationAction label="CashGames" icon={<TableBarIcon />} />
+        <BottomNavigationAction label="Tournament" icon={<EmojiEventsIcon />} />
+      </BottomNavigation>
```

Comment:
You don't really need any custom hooks to use this. What you're trying to do here is connect the BottomNav up with your react-router-dom state. Look through the react-router-dom docs for a hook that gives you access to the current location path. That's what you want to use for `value` here (e.g. '/bankroll').

Additionally, each BottomNavAction accepts a `value` prop that should be set to the page path. That way when the BottomNav `value` matches the Action `value`, the action will be highlighted as active in the UI.

That BottomNavAction `value` is also what will be passed into the BottomNav `onChange` as `newValue`. Look through the react-router-docs again for a hook that gives you the ability to navigate to a new path.

Reply:
imported useLocation, still need to look more into how to properly implement it in this instance

Reply:
looks like useParams might fit what i need better

Reply:
Both useful in different situations. Try logging each result to see what properties they give you access to. See where you can find those path values you need.